docs/security/tls_setup.md

```diff
@@ -0,0 +1,145 @@
+# TLS support for Percona Everest
+
+The Percona Everest server now supports Transport Layer Security (TLS)  for all incoming connections. TLS **encrypts** client and server communication. This keeps data safe from being **intercepted or changed**. Administrators can configure server certificates and private keys to enable **secure HTTPS access** to the Everest API server. This strengthens security for production environments.
+
+!!! note
+    - When TLS is enabled, the default server port (8080) will only accept `https` traffic, and `http` traffic will be rejected.
+    - We do not support redirects from `http` to `https`.
+    - Self-signed certificates aren't trusted by most browsers, so it's best to use a trusted certificate from a Certificate Authority (CA) in production.
+
+
+
+## TLS setup with Percona Everest
+
+!!! info "Important"
+    This section provides an example using **Helm**. You can also use the provided options with `everestctl` by using the flag `—helm.set`.
+
+
+### Use Cert-manager (recommended)     
+
+#### Prerequisites
+
+- Ensure that [cert-manager](https://cert-manager.io/docs/) has been deployed in your infrastructure.
+
+- Ensure that you have a properly configured **Issuer** or **ClusterIssuer** in place. See the [documentation](https://cert-manager.io/docs/configuration/) for details.
+
+
+#### Set up Percona Everest using cert-manager
+
+Here are the steps to set up the Percona Everest server using cert-manager:
+{.power-number}
+
+1. Install cert-manager:
+
+    ```sh
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.17.0/cert-manager.yaml
+    ```
+
+2. Create a configuration file named `values.yaml`:
+
+    ```sh
+    server:
+	tls: 
+	  enabled: true
+	  certificate:
+	  create: true
+	  domain: example.com
+	  issuer:
+			group: cert-manager.io
+			kind: ClusterIssuer 
+			name: your-cluster-issuer
```

Comment:
The indentation in the rendered web page is not correct. `tls` needs to be under `server`. See screenshot below:

![Screenshot 2025-04-29 at 1 43 50 PM](https://github.com/user-attachments/assets/450288b1-0157-495b-8c91-2bf99a8c2ca4)

Reply:
@rasika-chivate this isn't fixed